lib/sles4sap_publiccloud.pm

```diff
@@ -369,13 +370,14 @@ sub stop_hana {
         # Crash needs to be executed as root and wait for host reboot
         $self->{my_instance}->wait_for_ssh(timeout => $timeout);
         $self->{my_instance}->run_ssh_command(cmd => "sudo su -c sync", timeout => "0", %args);
-        # Close SSH mux file before Crash test
-        $self->{my_instance}->run_ssh_command(cmd => " ", timeout => "0", ssh_opts => "-O exit");
+        # Try only extending ssh_opts
```

Comment:
It also make sense to document what `ServerAliveInterval=2` means

Reply:
It was there before, I'm keeping it. We can ask @lpalovsky for documentation next week.